File: src/java/1.groundZeroManager/Main.java
import java.util.ArrayList;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        System.out.println(
                "Hallo Nutzer Willkommen beim ersten Projekt einen Aufgaben Manager\n Bitte gib dein Alter und deinen Namen an");
        Scanner in = new Scanner(System.in);
        System.out.print("Name:");
        String name = in.nextLine();
        System.out.print("Alter:");
        int alter = in.nextInt();
        int neu = alter+ 5;
        System.out.println("In 5 jahren wirst du " +name +": " + neu + " Jahre alt sein!\n");

        ArrayList<String> aufgaben = new ArrayList<>();
        in.nextLine();
        while (true) {
            System.out.println("1: Aufgabe hinzufügen\n2: Alle Afugaben anzeigen\n3: Programm beenden");
            String job = in.nextLine();
            if (job.equals("1")) {
                System.out.println("Gebe die Aufgabe ein die Hinzugefügt werden soll:");
                aufgaben.add(in.nextLine());
            } else if (job.equals("2")) {
                int anzahl = 1;
                for (String s : aufgaben) {
                    System.out.println("Aufgabe " + anzahl + " ist " + s);
                    anzahl++;
                }

            } else if (job.equals("3")) {
                break;
            } else {
                System.out.println("Falsche Eingabe");
            }
        }
        in.close();
    }
}

File: src/python/1.groundZeroManager/main.py
def main():
    print("Hallo und Herzilich Willkommen in der python hölle\n bitte gebe deinen Namen und dein Alter ein")
    name = input("Name: ")
    alter =int(input("Alter: "))
    neu = alter+5
    print(f"in 5 Jahren wirst du {name}: {neu} Jahre alt sein")

    aufgaben =[]
    while(True):
        job = int(input("1: Neue Aufgabe hinzufügen\n2: Alle Anzeigen\n3: programm beenden "))
        if job == 1:
            aufgaben.append(input("Neue Aufgabe:"))
        elif job == 2:
            for i, aufgabe in enumerate(aufgaben, 1):
                print(f"Aufgabe {i} ist {aufgabe}")
        elif job == 3:
            break
        else:
            print("flasche eingabe")

if __name__ == "__main__":
    main()